src/relay/pass/type_infer.cc

```diff
@@ -311,14 +335,20 @@ class TypeInferencer : private ExprFunctor<Type(const Expr&)> {
 
     if (type_arity != number_of_args) {
```

Comment:
Is there a way we can also print the function source code in these error cases?

Reply:
What function source code? we could annotate a second error on the function using `ReportError`. I think changes in the quality of errors should be done in follow up work.

Reply:
This type error may be fixed either by changing the call site or the function definition, but we currently only print the call site. We can address it in another pr.